static/src/javascripts/projects/common/modules/email/run-checks.js

```diff
@@ -43,7 +43,8 @@ define([
 
     function userIsInAClashingAbTest() {
         var clashingTests = [
-            ['ParticipationStarRatings','star-rating']
+            ['ParticipationStarRatings','star-rating'],
```

Comment:
What is this used for? It looks as though this function is only called by `allEmailCanRun` - what does that do?